### PR TITLE
Atanas/grok 12136 bio msa progress

### PR DESCRIPTION
**Pull request by tanas80**
Add progress bar for MSA from top menu

### CODE REVIEW

File: packages/Bio/src/utils/multiple-sequence-alignment-ui.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import {ALPHABET, NOTATION} from '@datagrok-libraries/bio/src/utils/macromolecule';
+import {runKalign} from './multiple-sequence-alignment';
+import {pepseaMethods, runPepsea} from './pepsea';
+import {checkInputColumnUI} from './check-input-column';
+
+export function multipleSequenceAlignmentUI(col: DG.Column<string> | null = null) {
+  const table = col?.dataFrame ?? grok.shell.t;
+  const seqCol = col ?? table.columns.bySemType(DG.SEMTYPE.MACROMOLECULE);
+  if (seqCol == null) {
```
Comment (LaykDimon) [inline packages/Bio/src/utils/multiple-sequence-alignment-ui.ts:13]: strict comparison is preferred

File: packages/Bio/src/utils/multiple-sequence-alignment-ui.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  const inputRootStyles = [methodInput.root.style, gapOpenInput.root.style, gapExtendInput.root.style];
+
+  const colInput = ui.columnInput('Sequence', table, seqCol, () => {
+    const potentialCol = colInput.value;
+    const unusedName = table.columns.getUnusedName(`msa(${potentialCol.name})`);
+
+    if (checkInputColumnUI(
+      potentialCol, potentialCol.name, [NOTATION.FASTA], [ALPHABET.DNA, ALPHABET.RNA, ALPHABET.PT], false)) {
+      for (const inputRootStyle of inputRootStyles)
+        inputRootStyle.display = 'none';
```
Comment (LaykDimon) [inline packages/Bio/src/utils/multiple-sequence-alignment-ui.ts:34]: maybe set the class on these inputs where you will set the display to none?

File: packages/Bio/src/utils/multiple-sequence-alignment-ui.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      performAlignment = () => runKalign(potentialCol, false, unusedName, clustersColInput.value);
+    } else if (checkInputColumnUI(potentialCol, potentialCol.name, [NOTATION.HELM], [], false)) {
+      for (const inputRootStyle of inputRootStyles)
+        inputRootStyle.display = 'initial';
+
+      performAlignment = () => runPepsea(potentialCol, unusedName, methodInput.value!, gapOpenInput.value!,
+        gapExtendInput.value!, clustersColInput.value);
+    } else {
+      for (const inputRootStyle of inputRootStyles)
+        inputRootStyle.display = 'none';
```
Comment (LaykDimon) [inline packages/Bio/src/utils/multiple-sequence-alignment-ui.ts:45]: same here

File: packages/Bio/src/utils/multiple-sequence-alignment-ui.ts (hunk 2 above)
Comment (tanas80) [reply packages/Bio/src/utils/multiple-sequence-alignment-ui.ts:34]: class div.ui-form.inputs-hidden input[field-set=1] with display: none;

field-set - attribute of an input element

set form div class with 'inputs-hidden'

**Pull request merged**